libursa/DatabaseSnapshot.cpp

```diff
@@ -12,14 +12,12 @@ DatabaseSnapshot::DatabaseSnapshot(
     fs::path db_name, fs::path db_base,
     std::map<std::string, OnDiskIterator> iterators,
     std::vector<const OnDiskDataset *> datasets,
-    const std::map<uint64_t, std::unique_ptr<Task>> &tasks,
-    size_t max_memory_size)
+    const std::map<uint64_t, std::unique_ptr<Task>> &tasks)
```

Comment:
```suggestion
    const std::unordered_map<uint64_t, std::unique_ptr<Task>> &tasks)
```

This requires to `#include <unordered_map>`!